### ISSUE TITLE
[BUG - Release] BPModLoaderMod doesn't load

### ISSUE THREAD

**Issue by Reokin** [UE4SS-RE/RE-UE4SS#335]
**Branch or Release**
The Tag for the Release you are using, or the commit you used if you built from source yourself.
UE4SS_Xinput_v2.5.2
UE4SS_v2.5.2-605-g92624ab

**Game and Engine Version**
Game where the bug occurred, and the UE engine version of the game.
Escape the Backrooms
4.27.2

**Describe the bug**
Throws an error into the console and doesn't load logicmods

**To Reproduce**
1. Enable BPModLoaderMod
2. Launch game
3. See error in log

**Expected behavior**
To load mods

**Screenshots and UE4SS Log**
If applicable, add screenshots to help explain your problem, and upload the log.
```
[Lua::execute_file] lua_pcall returned No mapping for the Unicode character exists in the target multi-byte code page.
stack traceback:
	[C]: in function 'IterateGameDirectories'
	...ooms\Binaries\Win64\Mods\BPModLoaderMod\scripts\main.lua:115: in local 'LoadModConfigs'
	...ooms\Binaries\Win64\Mods\BPModLoaderMod\scripts\main.lua:163: in main chunk
```

![image](https://github.com/UE4SS-RE/RE-UE4SS/assets/58398966/dc0b24cc-f950-408e-a196-2eeffc969adf)

**Desktop (please complete the following information):**
 - OS: [e.g. Win10, Win11]
Windows 11 Pro 22H2 Build 22621.2338

**Additional context**

**Comment by UE4SS** [comment]
Do you have any unicode characters in the path to the game ? (this path: C:\Program Files\Steam\...)

**Comment by Reokin** [comment]
No, path to the game is E:\Steam\steamapps\common\EscapeTheBackrooms\EscapeTheBackrooms\Binaries\Win64

**Comment by asheliahut** [comment]
I have run into the same issue with PalWorld with my drive location being `F:\SteamLibrary\steamapps\common\Palworld\Pal\Binaries\Win64\`

**Comment by narknon** [comment]
This is fixed in #884. An experimental release should be available soon.
